rsconnect/bundle.py

```diff
@@ -1219,7 +1222,8 @@ def make_quarto_manifest(
     else:
         # Standalone Quarto document
         base_dir = dirname(file_or_directory)
-        relevant_files = [file_or_directory] + extra_files
+        file_name = basename(file_or_directory)
+        relevant_files = [file_name] + extra_files
```

Comment:
For standalone quarto files deployed by specifying a file path, the manifest should include the base name, not he full path filename.